rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/packages/PackageController.java

```diff
@@ -228,25 +232,31 @@ public ResponseEntity<?> getPackagesForUser(
         String queryString = request.getQueryString();
         Map<String, String> params = restControllerHelper.parseQueryString(queryString);
         boolean isSearchByName = name != null && !name.isEmpty();
-        boolean isSearchByType = CommonUtils.isNotNullEmptyOrWhitespace(packageManager);
+        boolean isSearchByPackageManager = CommonUtils.isNotNullEmptyOrWhitespace(packageManager);
+        boolean isSearchByVersion = version != null && !version.isEmpty();
```

Comment:
Cant we use the same `CommonsUtils.isNotNullEmptyWhitespace(_)` as in line number `235`